src/crimson/osd/pg_recovery.cc

```diff
@@ -527,10 +532,14 @@ void PGRecovery::enqueue_push(
   if (!added)
     return;
   peering_state.prepare_backfill_for_missing(obj, v, peers);
+  ongoing_pushes++;
```

Comment:
We should probably log `ongoing_pushes` (increases/decreases) somewhere to keep track of it